containers/codespaces-linux/.devcontainer/base.Dockerfile

```diff
@@ -68,8 +68,13 @@ RUN apt-get update && export DEBIAN_FRONTEND=noninteractive \
     && mkdir -p /usr/local/etc/vscode-dev-containers/ \
     && mv -f /tmp/scripts/first-run-notice.txt /usr/local/etc/vscode-dev-containers/
 
+# Remove existing Python installation from Oryx base image
+RUN rm -rf /opt/python && rm "${PYTHON_ROOT}/current"
+
 # Install Python, JupyterLab, common machine learning packages, and Ruby utilities
-RUN bash /tmp/scripts/python-debian.sh "none" "/opt/python/latest" "${PIPX_HOME}" "${USERNAME}" "true" \
+RUN bash /tmp/scripts/python-debian.sh "3.10.4" "/opt/python/3.10.4" "${PIPX_HOME}" "${USERNAME}" "true" "true" "false" \
+    # Preserve symbolic link from Oryx base image
+    && ln -sf /opt/python/3.10.4 "${PYTHON_ROOT}/current" \
```

Comment:
> We should also install `3.9.7` to keep the current kitchensink as is.

Good idea. I'll add a second call to the installer script to install `3.9.7` to `/opt/python/3.9.7`.

> It might be nicer to add the addition sym links

How does the new feature handle symlinks?

Reply:
> How does the new feature handle symlinks?

haha. It doesn't add `3 3.10 3.9 etc` but again that's a new way of how kitchensink is built with features.